tests/datatypes/test_list.py

```diff
@@ -139,6 +139,146 @@ def test_slice(self):
+
+
+    def test_list_list_comparisons(self):
+
+
+        # `this` (left list) is empty.
+
+        self.assertCodeExecution("""
+        print('>>> x = []')
+        x = []
+        print('>>> y = [1,2,3]')
+        y = [1,2,3]
+        print('>>> x > y')
+        x>y
+        print('>>> x >= y')
+        x >= y
+        print('>>> x < y')
+        x < y
+        print('>>> x <= y')
+        x <= y
+        """)
+
+        # `other` (right list) is empty
+
+        self.assertCodeExecution("""
+        print('>>> x = [1,2,3]')
+        x = [1,2,3]
+        print('>>> y = []')
+        y = []
+        print('>>> x > y')
+        x>y
+        print('>>> x >= y')
+        x >= y
+        print('>>> x < y')
+        x < y
+        print('>>> x <= y')
+        x <= y
+        """)
+
+        # both lists are empty
+        self.assertCodeExecution("""
+        print('>>> x = []')
+        x = []
+        print('>>> y = []')
+        y = []
+        print('>>> x > y')
+        x>y
+        print('>>> x >= y')
+        x >= y
+        print('>>> x < y')
+        x < y
+        print('>>> x <= y')
+        x <= y
+        """)
+
+        # `this` (left list) is shorter
+
+        self.assertCodeExecution("""
+        print('>>> x = [1,2]')
+        x = [1,2]
+        print('>>> y = [1,2,3]')
+        y = [1,2,3]
+        print('>>> x > y')
+        x>y
+        print('>>> x >= y')
+        x >= y
+        print('>>> x < y')
+        x < y
+        print('>>> x <= y')
+        x <= y
+        """)
+
+        # `other` (right list) is shorter
+
+        self.assertCodeExecution("""
+        print('>>> x = [1,2,3]')
+        x = [1,2,3]
+        print('>>> y = [1,2]')
+        y = [1,2]
+        print('>>> x > y')
+        x>y
+        print('>>> x >= y')
+        x >= y
+        print('>>> x < y')
+        x < y
+        print('>>> x <= y')
+        x <= y
+        """)
+
+
+        # comparable items aren't equal
+
+        self.assertCodeExecution("""
+        print('>>> x = [1,2]')
+        x = [1,2]
+        print('>>> y = [1,3]')
+        y = [1,3]
+        print('>>> x > y')
+        x>y
+        print('>>> x >= y')
+        x >= y
+        print('>>> x < y')
+        x < y
+        print('>>> x <= y')
+        x <= y
+        """)
+
+        self.assertCodeExecution("""
+        print('>>> x = [1,3]')
+        x = [1,3]
+        print('>>> y = [1,2]')
+        y = [1,2]
+        print('>>> x > y')
+        x>y
+        print('>>> x >= y')
+        x >= y
+        print('>>> x < y')
+        x < y
+        print('>>> x <= y')
+        x <= y
+        """)
+
+        # all items are equal
+
+        self.assertCodeExecution("""
+         print('>>> x = [1,2,3]')
+         x = [1,2,3]
+         print('>>> y = [1,2,3]')
+         y = [1,2,3]
+         print('>>> x > y')
+         x>y
+         print('>>> x >= y')
+         x >= y
+         print('>>> x < y')
+         x < y
+         print('>>> x <= y')
+         x <= y
+         """)
+
```

Comment:
thanks, will do. where is the proper place to implement a new assertion like `assertOrdering`? A method in `TranspileTestCase`? Some other existing class? A new class? If a new class, what file should it go in?

Regarding printing the result, you mentioned that before and I clearly didn't get it. Sorry about that.